main/webapp/modules/core/langs/translation-en.json

```
@@ -443,6 +443,7 @@
     "core-recon/cell-contains": "Each cell contains:",
     "core-recon/max-candidates": "Maximum number of candidates to return",
     "core-recon/service-title": "Services",
+    "core-recon/url-already-registered":"Service URL already exists.",
```

Maybe the error message could be a bit more user-centric, such as:
```suggestion
    "core-recon/url-already-registered":"This reconciliation service already exists.",
```